web/vue.config.js

```diff
@@ -28,67 +28,71 @@ RelativeBundleTracker.prototype.writeOutput = function(compiler, contents) {
 };
 
 /*** Vue config ***/
-let devServerHost = process.env.DOCKERIZED ? '0.0.0.0' : '127.0.0.1';
+let devServerHost = process.env.DOCKERIZED ? "0.0.0.0" : "127.0.0.1";
 
 let vueConfig = {
-  outputDir: 'static/dist',
+  outputDir: "static/dist",
 
   // When running `npm run serve`, paths in webpack-stats.json will point to the live dev server.
   // Otherwise they will point to the compiled assets URL.
-  publicPath: devMode ? 'http://localhost:8080/static/dist' : '/static/dist',
+  publicPath: devMode ? "http://localhost:8080/static/dist" : "/static/dist",
 
   pages: {
-    application: 'frontend/pages/application.js',
-    rich_text_editor: 'frontend/pages/rich_text_editor.js',
-    main: 'frontend/pages/main.scss',
-    test: 'frontend/pages/test.js',
-    vue_app: 'frontend/pages/vue_app.js',
-    casebook_html: "frontend/pages/casebook_html.scss"
+    application: "frontend/pages/application.js",
+    rich_text_editor: "frontend/pages/rich_text_editor.js",
+    main: "frontend/pages/main.scss",
+    test: "frontend/pages/test.js",
+    vue_app: "frontend/pages/vue_app.js",
+    as_printable_html_styles: "frontend/pages/as_printable_html.scss",
+    as_printable_html: "frontend/pages/as_printable_html.js",
```

Comment:
This is the only material change here (Prettier got excited about normalizing the quotation marks and I just let it).